solidity/contracts/vault/TBTCVault.sol

```diff
@@ -70,7 +71,11 @@ contract TBTCVault is IVault, Ownable {
         _;
     }
 
-    constructor(Bank _bank, TBTC _tbtcToken) {
+    constructor(
+        Bank _bank,
+        TBTC _tbtcToken,
+        Bridge _bridge
+    ) TBTCOptimisticMinting(_bridge) {
```

Comment:
Yeah, see 2e84c185c28b4dc7731ea9df2f879d5a1819de81, please. We used `bank.bridge()` in `TBTCVault` to have a single update point in case replacing the entire Bridge is needed. Before optimistic minting, `TBTCVault` worked only with `Bank` and was not interested in how the `Bridge` worked. Hence, it was possible to do `bank.bridge()` to obtain the `Bridge` address for the redemption request. Nothing more was needed. With the optimistic minting, `TBTCVault` must inspect the deposit state, so updating Bridge is no longer that easy. That being said, we can refer to the `bridge` field from `TBTCOptimisticMinting` the `TBTCVault` inherits from.